downstairs/src/region.rs

```diff
@@ -1527,6 +1583,8 @@ impl Region {
     ) -> Result<Region> {
         options.validate()?;
 
+        Self::set_max_open_files_rlimit(&log)?;
```

Comment:
Maybe, though a failure to set this would only matter if the existing limit wouldn't be enough for the Region, which if we continued would be bad news (a runtime failure I think!). I put a check in 82d76f1 for this